runtime_scan/pkg/scanner/job_managment.go

```diff
@@ -49,17 +50,19 @@ func (s *Scanner) jobBatchManagement(scanDone chan struct{}) {
 	imagesCompletedToScan := &s.progress.ImagesCompletedToScan
 	s.Unlock()
 
-	// queue of jobs
+	// queue of scan data
```

Comment:
each scan data is related to a job

Reply:
its a queue of scan data though.. scan data is not related to a specific job currently

Reply:
why not related. each job created for each image and it's all managed in scanData (one per job) :
```
type scanData struct {
	imageHash                    string
	imageID                      string
	contexts                     []*imagePodContext // All the pods that contain this image hash
	scanUUID                     string
```